core/config.go

```diff
@@ -41,5 +43,32 @@ func registerConfig() error {
 		return err
 	}
 
+	if err := config.Register(&config.Option{
+		Name:           "Locale",
+		Key:            CfgLocaleKey,
+		Description:    "Configures the locale for the user interface. This mainly affects rendering of dates, currency and numbers. Note that the Portmaster does not yet support different languages.",
+		OptType:        config.OptTypeString,
+		ExpertiseLevel: config.ExpertiseLevelUser,
+		ReleaseLevel:   config.ReleaseLevelStable,
+		DefaultValue:   "en-GB",
+		PossibleValues: []config.PossibleValue{
+			{
+				Name:  "en-GB",
+				Value: "en-GB",
+			},
+			{
+				Name:  "en-US",
+				Value: "en-US",
+			},
+		},
```

Comment:
Would suggest:

Name: `Date and Number Format`
Options: `American Format`, `European Format`

This is a lot easier to understand and not _too_ wrong for the start. ;)

Later this could evolve into language+format, as the locale also includes the language.